cmake/CMakeLists.txt

```diff
@@ -901,10 +903,10 @@ if (onnxruntime_ENABLE_CPUINFO)
       message(FATAL_ERROR "universal binary is not supported for apple framework")
     endif()
   else()
-    if (onnxruntime_BUILD_WEBASSEMBLY)
-    set(CPUINFO_SUPPORTED FALSE)
+    if (onnxruntime_BUILD_WEBASSEMBLY AND NOT onnxruntime_USE_XNNPACK)
```

Comment:
@snnn What's the reason for this change? I see the following warning from CPUINFO so I'm not sure how XNNPACK enables using CPUINFO when we're doing a wasm build.

> CMake Warning at external/pytorch_cpuinfo/CMakeLists.txt:78 (MESSAGE):
  Target operating system "Emscripten" is not supported in cpuinfo.  cpuinfo
  will compile, but cpuinfo_initialize() will always fail.

Reply:
Added a few more comments to clarify. Xnnpack needs clog from cpuinfo but the ORT code won't use it in an wasm build.